doc/manual/en/config/widgets/index.rst

```diff
@@ -81,21 +81,21 @@ These widgets have a purely informative function, an operation is not possible.
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
 | Widget                                            | Description                                                                                                                           |
 +===================================================+=======================================================================================================================================+
-| :doc:`Image <image/index>`                        | Adds a static image or a camera image to the visu.                                                                                    |
+| :doc:`Image <image/index>`                        | Adds a static image or a live camera image to the visu.                                                                               |
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`Info <info/index>`                          | Adds a dynamic field to the Visu that can represent values from the BUS. (e.g. 14-byte text or counter headers)                       |
+| :doc:`Info <info/index>`                          | Adds a dynamic field to the visu that can represent values from the BUS. (e.g. 14-byte text or temperature measurements)              |
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`Line <line/index>`                          | Adds a horizontal line to the visu to divide the visual side into individual areas.                                                   |
+| :doc:`Line <line/index>`                          | Adds a horizontal line to the visu to divide the display into individual areas.                                                       |
 +---------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`RGB <rgb/index>`                            | With the RGB widget you can display a color in the Visu.                                                                              |
+| :doc:`RGB <rgb/index>`                            | With the RGB widget you can display a colour in the visu.                                                                             |
```

Comment:
not if we want to use american english or british english regarding color vs. colour. I would tend to use color, but I am not sure how its handled throughout the docs.